snapcraft/tests/test_plugin_catkin.py

```diff
@@ -233,41 +259,36 @@ def test_invalid_catkin_workspace_invalid_subdir(self):
         self.assertEqual(
             str(raised.exception),
             'Unable to find package path: "{}"'.format(os.path.join(
-                plugin.sourcedir, self.properties.source_subdir)))
+                plugin.sourcedir, self.properties.source_space)))
 
-    def test_invalid_catkin_workspace_subdir_same_as_source(self):
-        self.properties.source_subdir = '.'
+    def test_invalid_catkin_workspace_source_space_same_as_source(self):
+        self.properties.source_space = '.'
 
         # sourcedir is expected to be the root of the Catkin workspace. Since
-        # source_subdir was specified to be the same as the root, this should
+        # source_space was specified to be the same as the root, this should
         # fail.
         with self.assertRaises(RuntimeError) as raised:
             plugin = catkin.CatkinPlugin('test-part', self.properties)
             plugin.pull()
```

Comment:
You can make this a oneline then ;-)

`catkin.CatkinPlugin('test-part', self.properties).pull()`